src/blueprint/from_config.rs

```diff
@@ -428,7 +476,11 @@ fn process_field_within_type(
       return Valid::<Type>::validate_or(
         Valid::fail(format!(
           "Inline can't be done because of {} resolver at [{}.{}]",
-          next_field.http.as_ref().map(|_| "http").unwrap_or_else(|| "unsafe"),
+          {
+            let next_dir_http = next_field.http.as_ref().map(|_| "http");
+            let next_dir_const = next_field.const_field.as_ref().map(|_| "const");
+            next_dir_http.or(next_dir_const).unwrap_or("unsafe")
+          },
```

Comment:
Showing all the resolvers in one error will be a better experience for the user.

Reply:
What do you suggest error message might look like?

Reply:
```
type Query {
  post: Post @http(path: "/posts/1") @http(path: "/users/{{value.userId}}") @inline(path: ["user", "name"])
}

type Post {
  userId: Int
  user: User @http(path: "/users/{{value.userId}}")
}
```

@tusharmath, I could be wrong but from usecase, it looks like only one resolvable directive will be on the path?